roles/identity-management/manage-rh-sso-identities/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+rh_sso_host: "192.168.16.3"
+rh_sso_port: 8080
+rh_sso_port_list:
```

Comment:
This list doesn't make sense here. Should probably just delete as it seems like a carry-over from the config-rh-sso role.